### PR TITLE
Test battle thibault

### PR DESCRIPTION
**Pull request by ThibaultBardinetLanglois**
Add a battle page when you click on the battle button

### CODE REVIEW

**Review by StephaneSauvaget — DISMISSED** [review]
Just be careful you add ItemController  and battle.twig.html not necessary

File: src/Controller/GameController.php [hunk 1]
```diff
@@ -17,4 +19,19 @@ public function play()
...
+
+    public function battle()
+    {
+        $stercketUser = new Stercket();
+        $stercketEnnemy = new Stercket();
+        $stercketUser->initialise('player');
+        $stercketEnnemy->initialise('wood');
+        return $this->twig->render('Game/battle.html.twig', [
+            "stercketUser" => $stercketUser,
+            "healthStercketUser" => $stercketUser->getHealth(),
```
Comment (RomainClair) [inline src/Controller/GameController.php]: You already send the stercketUser to twig, use it to access it's health directly in twig

File: src/Controller/GameController.php [hunk 2]
```diff
@@ -17,4 +19,19 @@ public function play()
...
+            "stercketEnnemy" => $stercketEnnemy,
+            "healthStercketEnnemy" => $stercketEnnemy->getHealth(),
```
Comment (RomainClair) [inline src/Controller/GameController.php]: Same as above

File: src/Controller/GameController.php [hunk 3]
```diff
@@ -17,4 +19,19 @@ public function play()
...
         return $this->twig->render('Game/play.html.twig');
     }
+
+    public function battle()
+    {
+        $stercketUser = new Stercket();
+        $stercketEnnemy = new Stercket();
+        $stercketUser->initialise('player');
+        $stercketEnnemy->initialise('wood');
+        return $this->twig->render('Game/battle.html.twig', [
```
Comment (RomainClair) [inline src/Controller/GameController.php]: There is no battle.html.twig view in the Game directory

File: src/Controller/ItemController.php [hunk 4]
```diff
@@ -2,7 +2,9 @@
 
 namespace App\Controller;
 
+use App\Model\Stercket;
 use App\Model\ItemManager;
+use App\Model\StercketManager;
```
Comment (RomainClair) [inline src/Controller/ItemController.php]: Why do you need Stercket in the Item controller ?
Do you even still need the item controller anymore ?

File: src/View/Game/play.html.twig [hunk 5]
```diff
@@ -8,6 +8,7 @@
             <link rel="stylesheet" href="/assets/css/play.css">
     {% endblock %}
     {% block content %}
+        {{ include ('Game/battle.html.twig') }}
```
Comment (RomainClair) [inline src/View/Game/play.html.twig:11]: This view does not exists

**Review by RomainClair — CHANGES_REQUESTED** [review]
battle.css is invalid according to w3c

The Item/battle.html.twig should be in the Game directory.... i guess

File: src/Controller/GameController.php (hunk 1 above)
Comment (ThibaultBardinetLanglois) [reply src/Controller/GameController.php]: For the moment we need to keep an acces to the stercket health before the combat to test  a futur status method which will define a state for the stercket concerning its hp

**Review by AmanelGau — DISMISSED** [review]
seems ok

**Review by RomainClair — CHANGES_REQUESTED** [review]
Item/battle.html.twig is probably useless and should be deleted

**Review by AmanelGau — DISMISSED** [review]
file deleted

**Review by RomainClair — DISMISSED** [review]

**Review by AmanelGau — APPROVED** [review]
Conflit solved

**Review by RomainClair — APPROVED** [review]

**Pull request merged**